README.rst

```diff
@@ -10,10 +26,10 @@ SageMaker Experiments is an AWS service for tracking machine learning Experiment
 Concepts
 --------
 
-- Experiment: A collection of related Trials. Add Trials to an Experiment that you wish to compare together.
-- Trial: A description of a multi-step machine learning workflow. Each step in the workflow is described by a TrialComponent.
-- TrialComponent: A description of a single step in a machine learning workflow. 
-- Tracker: A Python context-manager for logging information about a single TrialComponent.
+- **Experiment**: A collection of related Trials. Add Trials to an Experiment that you wish to compare together.
+- **Trial**: A description of a multi-step machine learning workflow. Each step in the workflow is described by a TrialComponent.
```

Comment:
Nit: The working seems to imply that trial components in a trial has an ordering.

Reply:
updated